### PR TITLE
docs(aws-cognito): add guide for AWS Cognito integration

### PR DESCRIPTION
**Pull request by hassankhan**
Story details: https://app.clubhouse.io/cube-dev/story/64

### CODE REVIEW

**Review by rpaik — APPROVED** [review]

File: docs/content/Auth/AWS-Cognito.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+permalink: /security/jwt/aws-cognito
+category: Authentication & Authorization
+subCategory: Guides
+menuOrder: 2
+---
+
+## Introduction
+
+In this guide, you'll learn how to integrate AWS Cognito authentication with a
+Cube.js deployment. If you already have a pre-existing application on Auth0 that
```
Comment (keydunov) [inline docs/content/Auth/AWS-Cognito.md:12]: It mentions Auth0, I assume it should be Cognito?

File: docs/content/Auth/AWS-Cognito.md [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+## Configure Cube.js
+
+Now we're ready to configure Cube.js to use AWS Cognito. Go to your Cube.js
+project and open the `.env` file and add the following, replacing the values
+wrapped in `<>`.
+
+```dotenv
+CUBEJS_JWK_URL=https://cognito-idp.<AWS_REGION>.amazonaws.com/<USER_POOL_ID>/.well-known/jwks.json
+CUBEJS_JWT_AUDIENCE=<APPLICATION_URL>
+CUBEJS_JWT_ISSUER=https://<AUTH0-SUBDOMAIN>.auth0.com/
```
Comment (keydunov) [inline docs/content/Auth/AWS-Cognito.md:63]: Maybe it should be a different issuer here, not the Auth0 one?

File: docs/content/Auth/AWS-Cognito.md [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+  style="text-align: center"
+>
+  <img
+  src="https://raw.githubusercontent.com/cube-js/cube.js/master/docs/content/Auth/cognito-get-jwt-05.png"
+  style="border: none"
+  width="80%"
+  />
+</p>
+
+> Image here
```
Comment (keydunov) [inline docs/content/Auth/AWS-Cognito.md:153]: Is it a placeholder?

**Comment by keydunov** [comment]
@hassankhan Looks great! I left few comments.

Also, it seems something wrong with the order in the menu - 
Screen Shot 2021-03-02 at 3 47 35 PM

**Comment by hassankhan** [comment]
@keydunov Thanks for catching those issues 😅 Should all be fixed now 👍

**Review by keydunov — APPROVED** [review]

**Pull request merged**